### PR TITLE
Initial STIEBEL ELTRON documentation

### PR DESCRIPTION
**Pull request by fucm**
**Description:**
Add initial STIEBEL ELTRON documentation.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#21199

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/climate.stiebel_eltron.markdown [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+{% configuration %}
+name:
+  description: Displayed name of the unit.
+  required: false
+  type: string
+hub:
+  description: The name of the hub where this slave is located.
+  required: false
+  default: default
```
Comment (klaasnicolaas) [inline source/_components/climate.stiebel_eltron.markdown]: ?? What is the default 😉

**Review by klaasnicolaas — COMMENTED** [review]
The config variables do not match those in the parent PR.

In the python code I see `name` as required and` hub` as optional. In the documentation you have both set as optional.

File: source/_components/climate.stiebel_eltron.markdown (hunk 1 above)
Comment (fucm) [reply source/_components/climate.stiebel_eltron.markdown]: Yes, that's not very consistent. :-) It's now corrected and made optional.

File: source/_components/stiebel_eltron.markdown [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+description: "Instructions on how to integrate STIEBEL ELTRON integral ventilation and heat pump units into Home Assistant."
+date: 2019-02-23 16:30 +0200
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: stiebel_eltron.png
+ha_category:
+  - Climate
+ha_release: 0.91
```
Comment (klaasnicolaas) [inline source/_components/stiebel_eltron.markdown]:
```suggestion
ha_release: 0.92
```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_components/stiebel_eltron.markdown (hunk 2 above)
Comment (fucm) [reply source/_components/stiebel_eltron.markdown]: Done

**Comment by fucm** [comment]
home-assistant PR is ready to be merged. Can this be approved as well?

File: source/_components/stiebel_eltron.markdown [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+comments: false
+sharing: true
+footer: true
+logo: stiebel_eltron.png
+ha_category:
+  - Climate
+ha_release: 0.92
+ha_iot_class: "Local Polling"
+redirect_from:
+  - /components/climate.stiebel_eltron/
```
Comment (MartinHjelmare) [inline source/_components/stiebel_eltron.markdown]: We can remove this. We only need redirect if there were existing pages that we moved.

Comment (fucm) [reply source/_components/stiebel_eltron.markdown]: Removed

**Review by MartinHjelmare — DISMISSED** [review]

**Review by klaasnicolaas — DISMISSED** [review]

**Review by klaasnicolaas — APPROVED** [review]

**Comment by klaasnicolaas** [comment]
Thnx for the PR! 👍

**Pull request merged**